UndertaleModLib/Models/UndertaleRoom.cs

```diff
@@ -581,15 +572,15 @@ public class Background : UndertaleObject, INotifyPropertyChanged, IDisposable
         /// <remarks>
         /// This attribute is UMT-only and does not exist in GameMaker.
         /// </remarks>
-        public bool TiledHorizontally { get => TileX > 0; set { TileX = value ? 1 : 0; OnPropertyChanged(); } }
+        public bool TiledHorizontally { get => tileX > 0; set { tileX = value ? 1 : 0; OnPropertyChanged(); } }
 
         /// <summary>
         /// Indicates whether this <see cref="Background"/> is tiled vertically.
         /// </summary>
         /// <remarks>
         /// This attribute is UMT-only and does not exist in GameMaker.
         /// </remarks>
```

Comment:
This documentation would need adjustment. Yes, it is technically UMT-only, but is the only way to access tileY which does exist in Gamemaker